src/Exception/MollieException.php

```diff
@@ -4,4 +4,5 @@
 
 class MollieException extends \Exception
 {
+    const CUSTOMER_EXCEPTION = 0;
```

Comment:
an exception which has **0** is the default one - all custom exceptions should start from 1